fashify/src/pages/Auth/Signup/Signup.jsx

```diff
@@ -128,12 +116,18 @@ const SignUp = () => {
             >
               Create New Account
             </button>
-            <button
-              className='btn btn-link-secondary outlined-secondary text-align-center brd-rd-semi-sq'
-              onClick={() => navigate('/login')}
-            >
-              Already have an account
-            </button>
+            <div className='auth-footer'>
+              <p>
+                <span>Already have an account?</span>
+                <span
+                  className='auth-signup'
+                  onClick={() => navigate('/login')}
+                >
+                  {' '}
+                  sign in
+                </span>
```

Comment:
Here, just a suggestion, you can use Link, you should avoid use of navigate as much as possible